workspace-manager-clienttests/src/main/resources/servers/workspace-dev.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "workspace-dev",
+  "description": "Workspace Dev env.",
+
+  "samUri": "https://sam.dsde-dev.broadinstitute.org",
+  "workspaceManagerUri": "https://workspace.dsde-dev.broadinstitute.org",
```

Comment:
Can you add the appropriate Data Repo URL here also? Since WM will talk to Data Repo, I think it may be helpful for future tests. I think the URL you want is https://jade.datarepo-dev.broadinstitute.org

Reply:
Added Data Repo URL to servers json and updated `build.gradle` to include Data Repo Client library as well.